documentation/TechnicalDesignDocument.tex

```diff
@@ -208,7 +208,27 @@
     \section{Continous Intergration}
     In this section build automation and its constituent components will be discussed.
     How we applied it, the software used and the targets and platforms supported are among a few.
+    \newpage
 
+    %------------------------------------------------------------------------------------------------------------------------------------------------------- Continuing Development
+    \section{Continuing Development}
+    This section describes the plan on how to continue with development in the best way.
+    The team worked on quite some user stories from april 9th, to june 15th.
+    The following functionalities are most critical at the moment to start working on:
+    \myparagraph{Communication from the View to the Editor}
```

Comment:
These should be subsections (also for the two paragraphs below this one)

Reply:
Fixed